packages/shared/src/lib/core/layer-2/utils/getL2ToL1StorageDepositBuffer.ts

```diff
@@ -0,0 +1,32 @@
+import { nodeInfo } from '@core/network'
+import { DEFAULT_PROTOCOL } from '@core/network/constants'
+import { NetworkId } from '@core/network/types'
+import { SendFlowType } from '@core/wallet/enums'
+import { get } from 'svelte/store'
+
+const L2_TO_L1_STORAGE_DEPOSIT_BUFFER_BYTES: { [key in UnwrapSendFlow]: bigint } = {
+    [SendFlowType.TokenUnwrap]: BigInt(565),
+    [SendFlowType.NftUnwrap]: BigInt(35),
+}
+
+type UnwrapSendFlow = SendFlowType.TokenUnwrap | SendFlowType.NftUnwrap
+
+export function getL2ToL1StorageDepositBuffer(type: UnwrapSendFlow, network: NetworkId): bigint {
+    const actualVByteCost = get(nodeInfo)?.protocol.rentStructure.vByteCost
```

Comment:
should this be `getStardustNetwork(networkId)?.protocol.rentStructure.vByteCost`